spaces/S000074/properties/P000162.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000074
+property: P000162
+value: true
+refs:
+- doi: 10.1007/978-1-4615-7819-2
+  name: Rings of Continuous Functions (Gillman and Jerison)
+---
+
+The identity function $\text{Id}:X\to \mathbb{R}^2$, $\text{Id}(x) = x$ from {S000074} to {S000176} is a continuous bijection. Since every subspace of {S000176} is realcompact, {S000074} is realcompact from corollary 8.18 in {{doi:10.1007/978-1-4615-7819-2}}.
```

Comment:
continuous injection?

Reply:
Ah, yes, I was copy-pasting it and didn't notice.